docs/integrations/builtin/credentials/dynatrace.md

```diff
@@ -0,0 +1,28 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+title: Dynatrace credentials
+description: Documentation for the Dynatrace credentials. Use these credentials to authenticate Dynatrace in n8n, a workflow automation platform.
+contentType: integration
+---
+# Dynatrace credentials
+
+--8<-- "_snippets/integrations/builtin/credentials/cred-only-statement.md"
+
+## Prerequisites
+
+Create a [Dynatrace](https://www.dynatrace.com/signup/){:target=_blank .external-link} account.
+
+## Related resources
+
+Refer to [Dynatrace's API documentation](https://docs.dynatrace.com/docs/dynatrace-api){:target=_blank .external-link} for more information about authenticating with the service.
+
+This is a credential-only node. Refer to [Custom API operations](/integrations/custom-operations/) to learn more. View [example workflows and related content](https://n8n.io/integrations/dynatrace/){:target=_blank .external-link} on n8n's website.
```

Comment:
@Joffcom the link to the marketing site integrations gave me a 404. Is this the correct link: https://n8n.io/integrations/dynatrace-api/ or is the one you have here correct and just pending-dev?

Reply:
I didn't realise that was already there, I will get this updated thanks.